tests/gpts/github/test-config/github_positive_suite.ts

```diff
@@ -184,7 +183,28 @@ export const gitHubBasicGoldenPathTemplateTests = (gptTemplate: string) => {
                 }
             expect(image_index).not.toBe(undefined)
             } 
-        }, 900000)        
+        }, 900000)   
+        
+        /**
+         * verify if the ACS Scan is successfully done from the logs of task steps
+         */
+        it(`Check if ACS Scan is successful for ${gptTemplate}`, async ()=> {
+            type tasks = {name: string}
+            const pipelineRun = await kubeClient.getPipelineRunByRepository(repositoryName, 'push')
+            if (pipelineRun && pipelineRun.metadata && pipelineRun.metadata.name) {
+                const tasks = await kubeClient.getTaskRunsFromPipelineRun(pipelineRun.metadata.name)
+                let podName: string = pipelineRun.metadata.name + '-acs-image-scan-pod'
+                const pod_logs = await kubeClient.readContainerLogs(podName,developmentNamespace,'step-rox-image-scan')
+                const regex = new RegExp("\"result\":\"SUCCESS\"", 'i');
+                const result = regex.test(pod_logs)
+                if (result){
+                    console.log("The logs from acs-image-scan lists the step as success \n\n" + pod_logs )
```

Comment:
maybe you can move this after the expect - afterwards you dont have to use **if**